src/azure-cli/azure/cli/command_modules/profile/custom.py

```diff
@@ -143,6 +143,10 @@ def login(cmd, username=None, password=None, tenant=None, scopes=None, allow_no_
     if identity:
         if in_cloud_console():
             return profile.login_in_cloud_shell()
+
+        if username:
+            from azure.cli.core.breaking_change import print_conditional_breaking_change
+            print_conditional_breaking_change(cmd.cli_ctx, tag='ManagedIdentityUsernameBreakingChange')
```

Comment:
Nothing is printed:

![image](https://github.com/user-attachments/assets/b39434cd-b563-44be-a3a0-21b8cb82202d)

@ReaNAiveD, could you kindly look at it?